ArjunaJTS/jts/tests/classes/com/hp/mwtests/ts/jts/orbspecific/local/timeout/TerminationTest.java

```diff
@@ -124,8 +124,6 @@ public void test()
                 else
                     handle.rollback();
 
-                assertFalse(commit);
```

Comment:
This line should *not* be removed because if commit is true then we expect an exception so this assert will not be triggered. On the other hand if commit is false then we do not expect an exception so this assertFalse(commit) statement will succeed.